plyer/facades/audio.py

```diff
@@ -0,0 +1,56 @@
+class Audio(object):
+    '''Audio Facade.
+
+    Used for recording audio.
+    Use method `start` to start record and `stop` for stop recording.
+    For hear, what you have just recorded use method `play`.
+
+    Status will tell you about current job of the Audio.
```

Comment:
don't put a capital at the biggining of the sentence if it's an api name, it can be confusing. Also, for more clarity, using "`" around api names will make them more distinguishable.

`status` will tell you about the…